lib/iex/test/iex/helpers_test.exs

```diff
@@ -419,7 +419,14 @@ defmodule IEx.HelpersTest do
     cleanup_modules([:sample])
   end
 
-  test "pid helper" do
+  test "pid/1 helper" do
+    assert "#PID<0.32767.3276>" == capture_iex(~s[pid("0.32767.3276")])
+    assert "#PID<0.5.6>" == capture_iex(~s[pid("0.5.6")])
```

Comment:
I think it's okay to keep both assertions here (what if we hardcoded single value as return? :)), but have only one in examples. :bowtie: